renovate.json

```diff
@@ -2,6 +2,12 @@
   "extends": ["config:base"],
   "rebaseStalePrs": true,
   "ignoreDeps": ["react-docgen-typescript"],
+  "schedule": ["on Thursday every 4 weeks of the year starting on the 3rd week"],
+  "labels": ["PR: Internal :seedling:"],
+  "postUpgradeTasks": {
+    "commands": ["yarn format", "yarn install"],
```

Comment:
It's a bit strange to see these in this order. Why not `install` followed by `format`?

Reply:
That makes more sense. Updated.